starboard/testing/fake_graphics_context_provider.cc

```diff
@@ -158,7 +178,23 @@ void FakeGraphicsContextProvider::InitializeWindow() {
 }
 
 void FakeGraphicsContextProvider::InitializeEGL() {
-  display_ = EGL_CALL_SIMPLE(eglGetDisplay(EGL_DEFAULT_DISPLAY));
+  std::vector<const char*> enabled_features = {nullptr};
+  std::vector<const char*> disabled_features = {nullptr};
+  std::vector<SbEglAttrib> display_attribs = {
+      EGL_PLATFORM_ANGLE_DEVICE_TYPE_ANGLE,
+      EGL_PLATFORM_ANGLE_DEVICE_TYPE_EGL_ANGLE,
+      EGL_PLATFORM_ANGLE_TYPE_ANGLE,
+      EGL_PLATFORM_ANGLE_TYPE_OPENGLES_ANGLE,
+      EGL_FEATURE_OVERRIDES_ENABLED_ANGLE,
+      reinterpret_cast<SbEglAttrib>(enabled_features.data()),
+      EGL_FEATURE_OVERRIDES_DISABLED_ANGLE,
+      reinterpret_cast<SbEglAttrib>(disabled_features.data()),
+      EGL_NONE  // Terminate the attribute list
+  };
+  display_ = EGL_CALL_SIMPLE(eglGetPlatformDisplay(
+      EGL_PLATFORM_ANGLE_ANGLE, reinterpret_cast<void*>(EGL_DEFAULT_DISPLAY),
+      &display_attribs[0]));
```

Comment:
I think it's more semantically correct to use `display_attribs.data()` because it gives access to the `underlying array serving as element storage`.